templates/dnsupdate-rfc2136/0/docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+rfc2136dns:
+  image: rancher/external-dns:v0.4.2
+  expose:
+   - 1000
+  command:
+  - -provider
+  - RFC2136
```

Comment:
Name slug should be lowercase: `rfc2136`
See https://github.com/rancher/external-dns/blob/master/providers/rfc2136/rfc2136.go#L23